hazelcast/src/main/java/com/hazelcast/topic/impl/TopicProxy.java

```diff
@@ -36,9 +42,11 @@
 
     protected static final String NULL_MESSAGE_IS_NOT_ALLOWED = "Null message is not allowed!";
     protected static final String NULL_LISTENER_IS_NOT_ALLOWED = "Null listener is not allowed!";
+    private final SerializationService serializationService;
 
     public TopicProxy(String name, NodeEngine nodeEngine, TopicService service) {
         super(name, nodeEngine, service);
+        this.serializationService = nodeEngine.getSerializationService();
```

Comment:
I don't think we need the service anymore.

Reply:
Removed